site/app/templates/officeHoursQueue/QueueFilter.twig

```diff
@@ -10,7 +27,8 @@
     <a id="new_queue_rand_token" onclick="genRandCode('new_queue_token')" onkeypress="genRandCode('new_queue_token')" title="Generate random access code" aria-label="Generate random access code" tabindex="0" style="padding:.5rem;">
       <i class="fas fa-sync"></i>
     </a>
-    <button id="open_new_queue_btn" type="submit" class="btn btn-primary">Open New Queue</button>
+      <input type="text" id="regex" name="regex" placeholder="Contact Info Regex Pattern" aria-label="Regex" maxlength="2048">
```

Comment:
should the max length be validated on the backend as well?

Reply:
I don't think it's necessary in this case, honestly do you think there should be a max-length attribute at all ?

Reply:
I'm not sure.  I assumed that it was there to prevent some error with the maximum length allowable for the regex or something.  If there is no max length, and the DB doesn't have a max length, I don't see any reason why it can't be removed